docs/deploy/create_server.rst

```diff
@@ -115,49 +115,60 @@ If you couldn't select Ubuntu above, follow these additional steps:
 
       reboot
 
-2. Apply core changes
+2. Set up DNS records
+---------------------
+
+#. Create a new hostname DNS entry in `GoDaddy <https://dcc.godaddy.com/manage/OPEN-CONTRACTING.ORG/dns>`__
+
+All server hostnames should be in the following format ocpXX.open-contracting.org (e.g. ocp01, ocp02 etc).
+Each server should have a unique hostname so for each new server, increment the counter by one. 
+To see which hostnames are currently in use `refer to the salt roster config <https://github.com/open-contracting/deploy/blob/master/salt-config/roster>__`.
```

Comment:
Please put paragraph text on a single line, rather than multiple lines.

```suggestion
Hostnames follow the format ``ocp##.open-contracting.org`` (ocp01, ocp02, etc.). Increment the number by 1 for each new server, to ensure the hostname is unique and used only once. To determine the greatest number, refer to GoDaddy or the `salt-config/roster <https://github.com/open-contracting/deploy/blob/master/salt-config/roster>`__ file.
```